stylesheets/components/_dangerdanger.scss

```diff
@@ -0,0 +1,6 @@
+.dangerdanger {
```

Comment:
Not sure of the relationship between classes `dangerdanger`, `references-dangerdanger`, `site-header-dangerdanger`, `references`, and `site-header`. If there is a bunch of refactoring work to be done, that could be its own PR.

Reply:
This change allows me to separate layout properties that belong to the wrapper element and the visual part that belong to SVG itself. I guess better convention would be helpful, but this separation would make more value with this PR I feel.